tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -345,12 +351,18 @@ def test_sharing(self, cluster, friend_account_logged_in_docker_cluster_pk_ssh):
         resource_class_name = cluster.config().get("resource_type").capitalize()
         config = cluster.config()
 
+        expected_creds = (
+            "ssh-sky-key"
+            if isinstance(cluster._creds, SSHSecret)
+            else f'{config["name"]}-ssh-secret'
```

Comment:
how come we're not hard coding the ssh-secret name?  (or is this just for testing purposes?)

Reply:
just for testing - if the secret has a priv key path itll be the SkySecret `ssh-sky-key`, but if it has a password without a priv key then itll just be a generic secret type, but tied and named specifically to the cluster